### PR TITLE
Work from Igalia for WAM integration in Flounder + tinyproxy

### PR DESCRIPTION
**Pull request by jdapena**

### CODE REVIEW

File: recipes-wam/chromium/chromium53.inc [hunk 1]
```diff
@@ -7,7 +7,7 @@ DEPENDS_append = " af-main-native"
 # for bindings  af-binder is required.
 DEPENDS_append = " af-binder"
 
-SRC_URI = "git://github.com/webosose/${PN}.git;branch=@1.agl;protocol=https"
+SRC_URI = "git://github.com/Igalia/${PN}.git;branch=flounder;protocol=https"
```
Comment (jdapena) [inline recipes-wam/chromium/chromium53.inc]: This needs to be fixed so it uses again the webosose repositories. Also, the branch is @1.agl.flounder.

File: recipes-wam/wam/wam-tinyproxy_1.10.0.bb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+SUMMARY = "Lightweight http(s) proxy daemon"
+HOMEPAGE = "https://tinyproxy.github.io/"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=751419260aa954499f7abaabaa882bbe"
+
+SRC_URI = "git://github.com/Igalia/${PN}.git;branch=wam-proxy;protocol=https"
```
Comment (jdapena) [inline recipes-wam/wam/wam-tinyproxy_1.10.0.bb]: Should we move this to webosose?

File: recipes-wam/wam/wam.bb [hunk 3]
```diff
@@ -5,14 +5,14 @@ LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7ca
...
-DEPENDS = "qtbase glib-2.0 chromium53 wayland-ivi-extension virtual/libhomescreen libwindowmanager"
+DEPENDS = "qtbase glib-2.0 chromium53 wayland-ivi-extension libhomescreen libwindowmanager"
 
 PR="r0"
 
 PROVIDES += "virtual/webruntime"
 RPROVIDES_${PN} += "virtual/webruntime"
 
-SRC_URI = "git://github.com/webosose/${PN}.git;branch=@1.agl;protocol=https"
+SRC_URI = "git://github.com/Igalia/${PN}.git;branch=flounder;protocol=https"
```
Comment (jdapena) [inline recipes-wam/wam/wam.bb]: Make it point to the webosose version of the repo and the right branch @1.agl.flounder

File: recipes-wam/wam/wam-tinyproxy_1.10.0.bb (hunk 2 above)
Comment (abhijeetk) [reply recipes-wam/wam/wam-tinyproxy_1.10.0.bb]: It makes sense to move it to webosose.

Comment (jaragunde) [reply recipes-wam/wam/wam-tinyproxy_1.10.0.bb]: If we aren't going to add more patches to tinyproxy (I had the impression we would need more patching), we might not need the fork; we could build from the 1.10.0 release and add a couple patches in the recipe. The fork could be used to maintain those patches but we wouldn't build from it.

I've been considering to create a .bbappend for the tinyproxy recipe that already exists in meta-openembedded, but I don't think that would be possible; we cannot modify the inherited bbclasses and one of the points of the wam-tinyproxy recipe is preventing the installation as a systemd service. Additionally, that recipe is for an old version. I sent a patch to update it and it was [merged into meta-openembedded master](http://cgit.openembedded.org/meta-openembedded/commit/meta-networking/recipes-support/tinyproxy?id=fa1156b91a96103bb3f6b2d0943b81044d9ef1ee), but it's not available in FF.

To sum up, no changes proposed: let's move the tinyproxy fork and this recipe under the webosose umbrella, and reconsider what to do with the recipe when we start merging meta-agl-lge into meta-agl.

Comment (jdapena) [reply recipes-wam/wam/wam-tinyproxy_1.10.0.bb]: I think we are fine for now using the wam-tinyproxy repository. Eventually we could move to use patches in the recipe, but I'd still keep the repository to maintain the patches to be added to the recipe.

For now it is fine.
